lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -117,6 +117,45 @@ $notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotator
   .annotator-listing {
     padding: 0 !important;
     margin: 0 !important;
+
+    .annotator-widget {
+
+      // using annotatorJS triangle styling for adder
+      &:after {
+        @include transform(rotate(45deg));
+        @include left(10px);
+        position: absolute;
+        bottom: -($baseline/2);
+        display: block;
+        width: 16px;
+        height: 16px;
+        content: " ";
+        background: $white;
+        border-bottom: 1px solid $gray-l2;
+        border-right: 1px solid $gray-l2;
+      }
+    }
+  }
+
+  .annotator-editor {
+
+    .annotator-widget {
+
+      // using annotatorJS triangle styling for adder
+      &:after {
+        @include transform(rotate(45deg));
```

Comment:
Almost all of these properties are repeated in two places. Can you create a notes-bubble Sass extend in this file and extend it here?